conf/nginx/server-base.conf.jnj

```diff
@@ -338,3 +338,9 @@ location ~ ^/b1/socket(/?)(.*) {
     proxy_set_header Connection "upgrade";
     proxy_read_timeout 300;
 }
+
+location /userfiles/	{
+    proxy_pass http://web_app_server;
+    client_max_body_size 100M;
```

Comment:
Why did you choose `100M`?

Reply:
@josephschorr I believe Ivan mentioned that is what the UI currently states.

Reply:
Oh shoot... missed that. Anyway, @kurtismullins @ibazulic We should make this come in from a config variable (and have the config variable also used in the UI) so that each installation can choose how large they want it to be

Reply:
Because of this:

https://github.com/quay/quay/blob/3.3-release/static/js/directives/ui/file-upload-box.js#L22

I haven't found any other reference regarding uploading through the UI so I presumed this is the limit.